index.html

```diff
@@ -86,6 +88,53 @@
 				maxime mollitia necessitatibus nobis pariatur quae quaerat quo. Doloremque, dolores eos necessitatibus tempora
 				tenetur vero.
 			</ts-note>
+			<article>
+				<h2>Table</h2>
+				<ts-table
+					id="ts-table-demo"
+					source='{
+					"cols": [
+						{"value": "Context", 			"size":"large"},
+						{"value": "Type", 				"size":"small", 	"display":"center"},
+						{"value": "Participants", "size":"small"},
+						{"value": "New Activity"},
+						{"value": "Last Activity", "size":"medium", "display":"right"}
+					],
+					"data":
+					[
+						{"isSelected": true,"data": [
+							{"isMobile": true, "value":{"title":"Purchase Request #3213432", "description":"Tradeshift"}, "type":"document"},
+							{"value":{"status":"ok", "text":"internal"}, "type":"status"},
+							{"value":1000},
+							{"value":"Every little bunny has a habit that is funny. It doesnt matter where he goes he always wrinkles up his nose."},
+							{"isMobile": true, "value":"Yesterday, 15:01", "display":"right"}
+						]},
+						{"data":[
+							{"isMobile": true, "value":{"title":"Invoice #321", "description":"Tradeshift"}, "type":"document"},
+							{"value":{"status":"error", "text":"private"}, "type":"status"},
+							{"value":1000},
+							{"value":"4 new messages"},
+							{"isMobile": true, "value":"Yesterday, 15:01", "display":"right"}
+							]},
+						{"data":[
+							{"isMobile": true, "value":{"title":"Invoice #321", "description":"Tradeshift"}, "type":"document"},
+							{"value":{"status":"note", "text":"external"}, "type":"status"},
+							{"value":1000},
+							{"value":"4 new messages"},
+							{"isMobile": true, "value":"Yesterday, 15:01", "display":"right"}
+						]}
+					]
+					}'
+				>
+				</ts-table>
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (6) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)